### ISSUE TITLE
StatusCode: 401

### ISSUE THREAD

**Issue by idShura** [skazantsev/WebDavClient#11]
Cannot connect to OwnCloud
WebDAV response - StatusCode: 401, Description: Unauthorized

**Comment by skazantsev** [comment]
Hi,
The provided description is not enough to investigate the problem. Could you provide me with more details: steps to reproduce, actual result, expected result and the code that indicates the problem?

**Comment by idShura** [comment]
Hi, the example shows the working test data (login, pass):

```
        static async void test1()
        {
            WebDavClientParams clientParams = new WebDavClientParams();

            clientParams.BaseAddress = new Uri("http://mytest.fastcomet.site/ownCloud/remote.php/webdav/");
            clientParams.Credentials = new NetworkCredential("test.user1", "test2@!7");

            using (WebDavClient webDavClient = new WebDavClient(clientParams))
            {
                var r = await webDavClient.PutFile("test6.txt", File.OpenRead("test.txt"), "text/xml"); // upload a resource

                if (!r.IsSuccessful)
                    MessageBox.Show(r.ToString());
            }
        }
```

**Comment by skazantsev** [comment]
As @drbergeron pointed out in https://github.com/skazantsev/WebDavClient/issues/13, it is probably an issue with the UseDefaultCredentials parameter.

The bug is fixed in [version 2.1.0](https://github.com/skazantsev/WebDavClient/releases/tag/2.1.0) which is available in [NuGet](https://www.nuget.org/packages/WebDav.Client).

For versions <2.0.1 the solution is too set UseDefaultCredentials to false.
